test/example/test.ts

```diff
@@ -1,18 +1,19 @@
 /*
- * Tests to verify stub behaviors
+ * Example tests against stubs
```

Comment:
Are these tests against an external project or testing within itself?

Reply:
This is intended to be an example test/usage that would be used in an external project.  I'm not sure it belongs in this repository in the long run, and will probably remove it once I resolve #31.